### PR TITLE
[IA-4126] Autopause UI for Azure runtimes

### PR DESCRIPTION
**Pull request by mlilynolting**
Setup Azure runtimes with autopause! Can enable/disable autopause on prospective runtimes, submit to create with the selected autopause value, and watch as after X minutes your runtime pauses. Opening an existing runtime shows a disabled autopause section with the existing value, if any, or a grayed checkbox with the input hidden otherwise.

Behavior validated locally & via unit tests.

merge BLOCKED until https://broadworkbench.atlassian.net/browse/IA-4168 merged.

### CODE REVIEW

File: src/pages/workspaces/workspace/analysis/modals/AzureComputeModal.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ export const AzureComputeModalBase = ({
   const [viewMode, setViewMode] = useState(undefined)
   const [currentRuntimeDetails, setCurrentRuntimeDetails] = useState(() => getCurrentRuntime(runtimes))
   const [computeConfig, setComputeConfig] = useState(defaultAzureComputeConfig)
-  const updateComputeConfig = (key, value) => setComputeConfig(_.set(key, value, computeConfig))
+  const updateComputeConfig = (key, value) => setComputeConfig({ ...computeConfig, [key]: value })
```
Comment (n8landolt) [inline src/pages/workspaces/workspace/analysis/modals/AzureComputeModal.js:39]: For other reviewers: I suggested this change - it's a bit easier to read and understand what's going on.

There's a bit of cleverness that goes on with _.set() that's additionally used in ComputeModal, which we will want to change as well.

File: src/pages/workspaces/workspace/analysis/modals/ComputeModal.js [hunk 2]
```diff
@@ -1151,6 +1151,7 @@ export const ComputeModalBase = ({
           'Learn more about autopause.',
           icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })
         ]),
+        // end Autopause component
```
Comment (n8landolt) [inline src/pages/workspaces/workspace/analysis/modals/ComputeModal.js:1154]: I mislead you on this comment - the NumberInput component should be included as well

**Review by n8landolt — APPROVED** [review]

**Review by n8landolt — CHANGES_REQUESTED** [review]
Seems like starting from clicking "Open" is calling the wrong endpoint.

**Review by n8landolt — APPROVED** [review]

**Review by LizBaldo — APPROVED** [review]
Tested on my BEE and it looks good!

**Pull request merged**